front-end/src/ducks/utils/api.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
 		return axios.get(process.env.REACT_APP_CHECK_AUTH, {withCredentials: true})
 	},
 	getUserEdit: () => {
-		return axios.get(process.env.REACT_APP_GET_USER_EDIT, {withCredentials: true})
+		return axios.get(process.env.REACT_APP_AUTH_USER_EDIT, {withCredentials: true})
```

Comment:
I think `REACT_APP_GET_USER_EDIT` is more descriptive of what that particular endpoint does. We should change our `.env` files rather than the reference in the code. What are your thoughts on that?

Reply:
I like GET_USER_EDIT way better. The only reason why I had it changed back to AUTH was due to the fact that we had it in our .env's like that. But I agree with you, lets change it.

Reply:
Cool, sounds good!